src/lib/analytics/analytics.js

```diff
@@ -409,21 +409,15 @@ const analytics = new class {
 
       const [logContext = {}] = bindings
       const [logMessage, eMsg = '', errorObj, extra = {}] = messages
+      const errorToPassIntoLog = new Error(`${logMessage}${eMsg ? ' ' + eMsg : ''}`)
```

Comment:
why it's needed ? we're loosing stack trace here

if we've got an error object logged - we have to send it exactly.
so the old solution when we just adjusted its message was the correct one

Reply:
i think better to revert it

Reply:
adjusted in my next commit